<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRadioamadorDadosComComentarios</class>
 <widget class="QDialog" name="frmRadioamadorDadosComComentarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnSalvar">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>12</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gravar alterações</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>96</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>103</y>
     <width>121</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Indicativo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtIndicativo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>102</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Modificado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtModificado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>103</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>DMRID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtDmrId">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>102</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Criado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCriado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>176</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>QRA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtQra"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>296</y>
     <width>281</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Cidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCidade"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>296</y>
     <width>281</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEstado"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>356</y>
     <width>281</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>País</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtPais"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>236</y>
     <width>281</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Celular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCelular"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>236</y>
     <width>281</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>E-Mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>176</y>
     <width>281</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Sobrenome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtSobreNome"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>176</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtNome"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>96</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Comentários: (Privado)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtComentarios"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnSalvar</tabstop>
  <tabstop>edtQra</tabstop>
  <tabstop>edtNome</tabstop>
  <tabstop>edtSobreNome</tabstop>
  <tabstop>edtEmail</tabstop>
  <tabstop>edtCelular</tabstop>
  <tabstop>edtCidade</tabstop>
  <tabstop>edtEstado</tabstop>
  <tabstop>edtPais</tabstop>
  <tabstop>edtComentarios</tabstop>
  <tabstop>edtIndicativo</tabstop>
  <tabstop>edtDmrId</tabstop>
  <tabstop>edtModificado</tabstop>
  <tabstop>edtCriado</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
